src/main.ts

```diff
@@ -100,13 +103,33 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
     var execOptions: any = {
         outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
         listeners: {
-            stdout: (data: any) => console.log(data.toString()), //to log the script output while the script is running.
-            errline: (data: string) => {
+            stdout: async (data: any) => {
+                let scannedResult = {result: null};
+                if(!config.credScanEnable) console.log(data.toString());
+                else if(!process.env.CREDSCAN) console.log(data.toString());
+                else if(config.credScanEnable || process.env.CREDSCAN){
+                    await cs.credscan(data.toString(), scannedResult);
+                    if(scannedResult.result)
+                    console.log(scannedResult.result);

```

Comment:
Indentation and use braces with if/else statement, even if it is of one line. It will help to not introduce bug in future.

Reply:
Oh, got it. Made the changes